lib/gooddata/bricks/middleware/logger_middleware.rb

```diff
@@ -19,23 +19,29 @@
 using NilExtensions
 
 require_relative 'base_middleware'
+require_relative 'mask_logger_decorator'
 
 module GoodData
   module Bricks
     class LoggerMiddleware < Bricks::Middleware
       def call(params)
         params = params.to_hash
-        logger = nil
         if params['GDC_LOGGING_OFF']
           logger = NilLogger.new
+        elsif params['log_directory'] && params['execution_id']
```

Comment:
info: Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy) (reek)